src/Microsoft.OData.Edm/ReferenceEqualityComparer.cs

```diff
@@ -0,0 +1,86 @@
+//---------------------------------------------------------------------
+// <copyright file="ReferenceEqualityComparer.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>

```

Comment:
is this class used anywhere?  how does it differ from the default ReferenceEquityComparer class?

Reply:
This was in existing code